.github/workflows/release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Draft release
+
+on:
+  workflow_dispatch:
```

Comment:
I decided for the first version of this to not add on new tags but we can add that in next version. Happy to be convinced to add it here though.

Reply:
Makes sense to me!